pages/[page].tsx

```diff
@@ -0,0 +1,68 @@
+import { GetStaticProps } from 'next';
+import { useRouter } from 'next/router';
+import React from 'react';
+import Layout from '../components/Layout';
+import styles from '../styles/Page.module.scss';
+import {Link, pageQuery, PageProps} from '../utils';
+
+export default function Page(props: PageProps): JSX.Element {
+  const {pageName, links, redirect} = props;
+  const router = useRouter();
+
+  if (redirect || (typeof window !== 'undefined' && router.isFallback)) {
```

Comment:
not sure this isFallback is correct

check out this [link](https://nextjs.org/docs/basic-features/data-fetching#fallback-true) to learn more about fallback

Reply:
lets just remove the router.isFallback!